src/studio/src/designer/backend/Startup.cs

```diff
@@ -143,6 +147,27 @@ public void Configure(IApplicationBuilder appBuilder, IWebHostEnvironment env)
                 appBuilder.UseExceptionHandler("/error");
             }
 
+            if (Configuration.GetValue<bool>("PostgreSQLSettings:EnableDBConnection"))
+            {
+                ConsoleTraceService traceService = new ConsoleTraceService { IsDebugEnabled = true };
+
+                string connectionString = string.Format(
+                    Configuration.GetValue<string>("PostgreSQLSettings:AdminConnectionString"),
+                    Configuration.GetValue<string>("PostgreSQLSettings:DesignerDbAdminPwd"));
+
+                appBuilder.UseYuniql(
+                    new PostgreSqlDataService(traceService),
+                    new PostgreSqlBulkImportService(traceService),
+                    traceService,
+                    new Yuniql.AspNetCore.Configuration
+                    {
+                        Workspace = Path.Combine(Environment.CurrentDirectory, Configuration.GetValue<string>("PostgreSQLSettings:WorkspacePath")),
+                        ConnectionString = connectionString,
+                        IsAutoCreateDatabase = false,
+                        IsDebug = true
```

Comment:
do we want this to be debug for all environments  or just dev/Locally?

Reply:
Removed tracelogging to console.